src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -26,6 +26,7 @@ public sealed partial class JsonDocument : System.IDisposable
         public static System.Threading.Tasks.Task<System.Text.Json.JsonDocument> ParseAsync(System.IO.Stream utf8Json, System.Text.Json.JsonDocumentOptions options = default(System.Text.Json.JsonDocumentOptions), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static System.Text.Json.JsonDocument ParseValue(ref System.Text.Json.Utf8JsonReader reader) { throw null; }
         public static bool TryParseValue(ref System.Text.Json.Utf8JsonReader reader, out System.Text.Json.JsonDocument document) { throw null; }
+        public void Write(Utf8JsonWriter writer) { }
```

Comment:
Where did the name here come from?  WriteValue seems more consistent with JsonElement.

Reply:
From the API review, we had agreed the API name be called `Write` here. The action being, I want to write a document. `WriteValue` on document seems strange to me.

Reply:
In the usability study review? I don't think we discussed the names at all, I looked at Immo's note as just shorthand for "yeah, we should add a write accelerator here".